examples/nodejs/lambda-examples/cloudwatch-metrics/infrastructure/bin/infrastructure.ts

```diff
@@ -0,0 +1,17 @@
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import {MomentoMetricsStack} from '../lib/momento-metrics-stack';
+
+const app = new cdk.App();
+new MomentoMetricsStack(app, 'MomentoMetricsMiddlewareExample', {
+  /* If you don't specify 'env', this stack will be environment-agnostic.
+   * Account/Region-dependent features and context lookups will not work,
+   * but a single synthesized template can be deployed anywhere. */
+  /* Uncomment the next line to specialize this stack for the AWS Account
+   * and Region that are implied by the current CLI configuration. */
+  // env: { account: process.env.CDK_DEFAULT_ACCOUNT, region: process.env.CDK_DEFAULT_REGION },
+  /* Uncomment the next line if you know exactly what Account and Region you
+   * want to deploy the stack to. */
+  // env: { account: '123456789012', region: 'us-east-1' },
```

Comment:
nit: looks like we are hardcoding the region here to `us-west-2`, so even if a user passes in a different env, we will still have resources in west-2. I don't think it would be hard to change this, or we can just remove this text and say it will only deploy to `us-west-2`

Reply:
good catch @bruuuuuuuce . let's see if we can get rid of the hard-coded us-west-2 refs